lib/events.js

```diff
@@ -338,7 +341,7 @@ EventEmitter.init = function(opts) {
 
   if (this._events === undefined ||
       this._events === ObjectGetPrototypeOf(this)._events) {
-    this._events = ObjectCreate(null);
+    this._events = new NullObject();
```

Comment:
This change is fairly problematic, I think. At the very least it's semver-major. Unfortunately we have a number of ecosystem modules that touch `this._events` directly and changing the type of thing that is can definitely break stuff.